assets/src/containers/FilesAccessed.js

```diff
@@ -9,9 +9,11 @@ import RangeSlider from '../components/RangeSlider'
 import FormControl from '@material-ui/core/FormControl'
```

Comment:
This commit takes care of the Files enable and disable view. I have simplified code a bit